.github/workflows/ok-to-preview.yml

```diff
@@ -0,0 +1,60 @@
+name: 🆗 Ok to preview
+
+on:
+  pull_request_target:
+    types: [labeled]
+
+jobs:
+  ok-to-preview:
+    if: ${{ github.event.label.name == 'ok-to-preview' }}
+    runs-on: ubuntu-latest
+    steps:
+
+    - name: Get changed files
+      #uses: lots0logs/gh-action-get-changed-files@2.1.4
+      uses: Mineflash07/gh-action-get-changed-files@feature/support-pr-target-event # Remove as soon as PR is merged
+      with:
+        token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
I don't think this workflow is relevant at the moment 😊

Reply:
Removed!